tests/php/RumTest.php

```
@@ -0,0 +1,286 @@
+				'path'  => '/',
+				'lcp'   => 3000,
+			)
+		);
+
+		$data  = RUM::get_data();
+		$today = gmdate( 'Y-m-d' );
+		$this->assertSame( 2, $data[ $today ]['/']['lcp']['n'] );
+		$this->assertSame( 4000.0, $data[ $today ]['/']['lcp']['sum'] );
+		$this->assertSame( 1000.0, $data[ $today ]['/']['lcp']['min'] );
+		$this->assertSame( 3000.0, $data[ $today ]['/']['lcp']['max'] );
+	}
```

PHPMD (2.15.0)

[error] test_collect_accumulates_samples accesses the super-global variable $_SERVER. (undefined)

(Superglobals)